.github/workflows/go.yml

```diff
@@ -12,13 +12,29 @@ permissions:
   contents: read
 
 jobs:
+  supportedVersions:
+    name: Fetch supported Go versions
+    runs-on: ubuntu-latest
+    outputs:
+        supported_versions: ${{ steps.matrix.outputs.supported_versions }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+      - name: Read supported_go_versions.txt
+        id: matrix
+        run: |
+            versions=$(head -n 3 supported_go_versions.txt)

```

Comment:
```suggestion

```

I feel like keeping a long list of versions in the `supported_go_versions` will be confusing in the future, i.e. I would expect support for all versions in that list.

Could we instead remove the latest version from this list when adding support to a new one?

Reply:
Agreed 💯 
I'll make the changes. Thanks for the suggestion!